crates/compiler/load_internal/src/file.rs

```diff
@@ -3167,33 +3172,43 @@ fn finish_specialization<'a>(
                     }
                 };
 
-                let symbol = match platform_data {
+                let exposed_symbols_and_layouts = match platform_data {
```

Comment:
We used to get exactly 1 host entrypoint here; now we pass along however many as the `platform` module specified.